Source/WebCore/platform/ScrollAlignment.h

```diff
@@ -42,6 +42,8 @@
  */
 
 #pragma once
+#include "ScrollLogicalPosition.h"
```

Comment:
This is a layering violation (ScrollLogicalPosition.h is in `/page`).

Reply:
Fixed